embedded-db2/src/test/resources/container-license-acceptance.txt

```diff
@@ -0,0 +1 @@
+ibmcom/db2:11.5.0.0a
```

Comment:
Should I remove this file? I must be here because of DB2 licensing

Reply:
so, keep it